tests/EnumTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\Enum\Tests;
+
+use Patchlevel\Enum\EnumException;
+use Patchlevel\Enum\Tests\Enums\Status;
+use PHPUnit\Framework\TestCase;
+
+class EnumTest extends TestCase
+{
+    public function testCreateEnum()
```

Comment:
`: void` as return type is missing. I think we should add some sort of cs tooling asap